jsk_perception/docker/ofa/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM pytorch/pytorch:1.8.1-cuda11.1-cudnn8-devel
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt -o Acquire::AllowInsecureRepositories=true update \
+    && apt-get install -y \
+    curl \
+    git \
+    libopencv-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+RUN git clone https://github.com/OFA-Sys/OFA \
+    && mkdir -p /var/mount
+RUN cd OFA \
+    && pip install -r requirements.txt \
+    && pip install "protobuf<3.20.0" "numpy>=1.20.3" flask
```

Comment:
```
[ INFO] [1680156383.962963126]: [/ofa_ros_client_rviz_p51s_30801_6669198759294921931] Stereo is NOT SUPPORTED
[ INFO] [1680156383.963053325]: [/ofa_ros_client_rviz_p51s_30801_6669198759294921931] OpenGL device: Quadro P500/PCIe/SSE2
[ INFO] [1680156383.963096437]: [/ofa_ros_client_rviz_p51s_30801_6669198759294921931] OpenGl version: 4.6 (GLSL 4.6).
Traceback (most recent call last):
  File "server.py", line 7, in <module>
    from fairseq import utils, tasks
  File "/workspace/OFA/fairseq/fairseq/__init__.py", line 32, in <module>
    import fairseq.criterions  # noqa
  File "/workspace/OFA/fairseq/fairseq/criterions/__init__.py", line 36, in <module>
    importlib.import_module("fairseq.criterions." + file_name)
  File "/opt/conda/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/workspace/OFA/fairseq/fairseq/criterions/ctc.py", line 18, in <module>
    from fairseq.data.data_utils import post_process
  File "/workspace/OFA/fairseq/fairseq/data/__init__.py", line 24, in <module>
    from .indexed_dataset import (
  File "/workspace/OFA/fairseq/fairseq/data/indexed_dataset.py", line 121, in <module>
    6: np.float,
  File "/opt/conda/lib/python3.8/site-packages/numpy/__init__.py", line 305, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
[ofa_api-2] process has finished cleanly
log file: /home/k-okada/.ros/log/fda89ed2-cec0-11ed-96cb-2b147920af86/ofa_api-2*.log
^C[ofa_ros_client_rqt_p51s_30801_7031696000715711632-5] killing on exit
[ofa_ros_client_rviz_p51s_30801_6669198759294921931-4] killing on exit
```
need to set `numpy>=1.20.3,<1.24.0` ? https://stackoverflow.com/questions/74844262/how-can-i-solve-error-module-numpy-has-no-attribute-float-in-python

Reply:
```
  File "/opt/conda/lib/python3.8/site-packages/lightning_fabric/__init__.py", line 29, in <module>
    from lightning_fabric.fabric import Fabric  # noqa: E402
  File "/opt/conda/lib/python3.8/site-packages/lightning_fabric/fabric.py", line 32, in <module>
    from lightning_fabric.plugins import Precision  # avoid circular imports: # isort: split
  File "/opt/conda/lib/python3.8/site-packages/lightning_fabric/plugins/__init__.py", line 18, in <module>
    from lightning_fabric.plugins.precision.amp import MixedPrecision
  File "/opt/conda/lib/python3.8/site-packages/lightning_fabric/plugins/precision/__init__.py", line 14, in <module>
    from lightning_fabric.plugins.precision.amp import MixedPrecision
  File "/opt/conda/lib/python3.8/site-packages/lightning_fabric/plugins/precision/amp.py", line 29, in <module>
    class MixedPrecision(Precision):
  File "/opt/conda/lib/python3.8/site-packages/lightning_fabric/plugins/precision/amp.py", line 96, in MixedPrecision
    def _autocast_context_manager(self) -> torch.autocast:
AttributeError: module 'torch' has no attribute 'autocast'
[ofa_api-2] process has finished cleanly

```
```
  File "/opt/conda/lib/python3.8/site-packages/datasets/arrow_dataset.py", line 64, in <module>
    from . import config
  File "/opt/conda/lib/python3.8/site-packages/datasets/config.py", line 134, in <module>
    importlib.import_module("soundfile").__libsndfile_version__
  File "/opt/conda/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/opt/conda/lib/python3.8/site-packages/soundfile.py", line 192, in <module>
    _snd = _ffi.dlopen(_explicit_libname)
OSError: cannot load library 'libsndfile.so': libsndfile.so: cannot open shared object file: No such file or directory
[ofa_api-2] process has finished cleanly
```

```
    from .speech_tasks import *
  File "/workspace/OFA/tasks/speech_tasks/__init__.py", line 1, in <module>
    from .unify_task import UnifySpeechTextTask
  File "/workspace/OFA/tasks/speech_tasks/unify_task.py", line 16, in <module>
    from data.s2t_data.unify_dataset import UnifyDataset
  File "/workspace/OFA/data/s2t_data/unify_dataset.py", line 33, in <module>
    import torchaudio
ModuleNotFoundError: No module named 'torchaudio'
[ofa_api-2] process has finished cleanly
```
```
  File "server.py", line 11, in <module>
    from utils.eval_utils import eval_step
  File "/workspace/OFA/utils/eval_utils.py", line 17, in <module>
    from tasks.nlg_tasks.gigaword import fix_tokenization
  File "/workspace/OFA/tasks/__init__.py", line 6, in <module>
    from .speech_tasks import *
  File "/workspace/OFA/tasks/speech_tasks/__init__.py", line 1, in <module>
    from .unify_task import UnifySpeechTextTask
  File "/workspace/OFA/tasks/speech_tasks/unify_task.py", line 16, in <module>
    from data.s2t_data.unify_dataset import UnifyDataset
  File "/workspace/OFA/data/s2t_data/unify_dataset.py", line 38, in <module>
    from g2p_en import G2p
ModuleNotFoundError: No module named 'g2p_en'
[ofa_api-2] process has finished cleanly
```